Tools/machines/perlmutter-nersc/perlmutter.sbatch

```diff
@@ -16,8 +16,6 @@
 #SBATCH -C gpu
 #SBATCH -c 32
 #SBATCH --ntasks-per-node=4
-#SBATCH --gpus-per-task=1
-#SBATCH --gpu-bind=single:1
 #SBATCH -o WarpX.o%j
```

Comment:
Need `-#SBATCH --gpus-per-node=4`, to make sure you're getting all 4 nodes to begin with.

Reply:
ok, though that might be default